TACC/Core-CMS
# Hotfix/texascale org/2020 launch

# Overview

Update Core with content that Texascale.org has long since—for half a year—been using as stable Core code.

# Changes

1. __Doc__: Add section to a CSS Readme.
2. __New__: Add Image Map component; as used on:
    - [Frontera Secret Sauce (dev)](https://texascale-dev.tacc.utexas.edu/2020/visualizing-science/frontera-secret-sauce/) ([prod](https://texascale.org/2020/visualizing-science/frontera-secret-sauce/)) 
    - [What's In a Node? (dev)](https://texascale.tacc.utexas.edu/2019/visualizing-science/whats-in-a-node/) ([prod](https://texascale.org/2019/visualizing-science/whats-in-a-node/))
3. `taccsite_custom`
    1. __New__: Remove borders from Texascale tables
    2. __New__: Load Core Image Map component for Texascale

# Testing

> @tacc-wbomar ran through these test steps. The only tweaks necessary were Changes 3.1 and 3.2, and those only affected these pages:
> - [Frontera Secret Sauce (dev)](https://texascale-dev.tacc.utexas.edu/2020/visualizing-science/frontera-secret-sauce/)
> - [What's In a Node? (dev)](https://texascale.tacc.utexas.edu/2019/visualizing-science/whats-in-a-node/)

- Load all pages on Dev and ensure the __not__-"DEPRECATED" templates are used.
- Load all pages on dev and ensure they still look like those on production.
